### ISSUE TITLE
Unable to load anything

### ISSUE THREAD

**Issue by Dhull2288** [Micke-K/IntuneManagement#208]
Whatever I click, it doesn't show up. Here is the error:

Invoke-MgGraphRequest : A parameter cannot be found that matches parameter name 'Url'.
At C:\temp\IntuneManagement-master\Extensions\MSGraph.psm1:619 char:41
+     $graphObjects = Invoke-GraphRequest -Url $url @params
+                                         ~~~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-MgGraphRequest], ParameterBindingExc 
   eption
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.Graph.PowerShell.Authentication. 
   Cmdlets.InvokeMgGraphRequest

**Comment by Micke-K** [comment]
Hello!

Are you running it from ISE, Visual Studio Code or something similar?

This can happen if Microsoft Graph module is installed and loaded which causes the code to run the Invoke-GraphRequest from the Graph module instead of IntuneManagement. Invoke-GraphRequest is also an alias for Invoke-MgGraphRequest.

Can you replace the attached psm1 file and try with that. Let me know how it goes.

Cheers!

[CloudAPIPowerShellManagement.zip](https://github.com/Micke-K/IntuneManagement/files/14836035/CloudAPIPowerShellManagement.zip)

**Comment by Dhull2288** [comment]
Yes, I was using ISE. Replacing the psm1 file fixed the problem.
Thank you so much for the help and the explanation.

**Comment by Micke-K** [comment]
Closing old issue.

**Comment by joshiajanimesh** [comment]
I'm getting this issue even when launching start-intuneManagement.PS1 from Powershell/Terminal. 
Please note my system has microsoft.Graph and microsoft.Graph.Beta modules installed.
